osidb/tests/test_flaw.py

```diff
@@ -914,3 +928,9 @@ def test_validate_embargoing_public_flaw(self):
             ]
             flaw.unembargo_dt = tzdatetime(2022, 1, 1)
             flaw.save()
+
+    def test_validate_flaw_without_affect(self):
+        """test that embargoed flaws without affect raises an error"""
```

Comment:
nitpick: I guess that the word `embargoed` is some leftover

Reply:
That is right, sorry, I will fix it.